Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -98,6 +101,23 @@ const Contest = () => {
         [],
     );
 
+    const renderParticipants = useCallback(
+        (participants: number, isActive: boolean) => (
+            <span>
+                {isActive
+                    ? 'Active'
+                    : 'Total'}
```

Comment:
Don't do this inside HTML, by convention. Calculate it inside a variable and print the variablee